packages/react-components/react-spinbutton/src/components/SpinButton/useSpinButtonStyles.styles.ts

```diff
@@ -25,7 +25,7 @@ const useRootClassName = makeResetStyles({
   gridTemplateRows: '1fr 1fr',
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31385&commitId=95e5a9ae2072b5b602428a4b071c7526a432f494&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - High Contrast.normal.chromium.png|0| Removed|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Field</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.SpinButton+horizontal.default.chromium.png|246| Changed|
|Field.SpinButton.default.chromium.png|1593| Changed|

</details>


  <details><summary><b>SpinButton Converged</b> 263 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverDecrement.chromium.png|767| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverInput.chromium.png|364| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownDecrement.chromium.png|199| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverDecrement.chromium.png|96| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverInput.chromium.png|22| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverDecrement.chromium.png|38| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverIncrement.chromium.png|96| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.rest.chromium.png|514| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.focused.chromium.png|277| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.focused.chromium.png|508| Changed|
|SpinButton Converged.Appearance- filled-darker.hoverDecrement.chromium.png|38| Changed|
|SpinButton Converged.Appearance- filled-darker.focused.chromium.png|277| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownIncrement.chromium.png|406| Changed|
|SpinButton Converged.Appearance- filled-darker.mouseDownDecrement.chromium.png|401| Changed|
|SpinButton Converged.Appearance- filled-darker.hoverInput.chromium.png|22| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownDecrement.chromium.png|401| Changed|
|SpinButton Converged.Appearance- filled-darker.rest.chromium.png|22| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverIncrement.chromium.png|768| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownIncrement.chromium.png|911| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownIncrement.chromium.png|199| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.focused.chromium.png|222| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverInput.chromium.png|19| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.rest.chromium.png|22| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverDecrement.chromium.png|765| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverIncrement.chromium.png|89| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverDecrement.chromium.png|89| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverIncrement.chromium.png|765| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownDecrement.chromium.png|160| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.focused.chromium.png|505| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownIncrement.chromium.png|908| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.rest.chromium.png|22| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverInput.chromium.png|22| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverInput.chromium.png|362| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverDecrement.chromium.png|40| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownDecrement.chromium.png|896| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownDecrement.chromium.png|419| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverInput.chromium.png|23| Changed|
|SpinButton Converged.Appearance- filled-darker.hoverIncrement.chromium.png|38| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownIncrement.chromium.png|161| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverIncrement.chromium.png|40| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.focused.chromium.png|184| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.focused.chromium.png|295| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.rest.chromium.png|19| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownDecrement.chromium.png|893| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.rest.chromium.png|23| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverIncrement.chromium.png|38| Changed|
|SpinButton Converged.Appearance- filled-lighter.hoverDecrement.chromium.png|40| Changed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownDecrement.chromium.png|420| Changed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownIncrement.chromium.png|425| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverInput.chromium.png|221| Changed|
|SpinButton Converged.Appearance- filled-lighter.hoverInput.chromium.png|23| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownIncrement.chromium.png|424| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.focused.chromium.png|255| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.focused.chromium.png|508| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverDecrement.chromium.png|767| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverInput.chromium.png|364| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownIncrement.chromium.png|230| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.rest.chromium.png|192| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownIncrement.chromium.png|911| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverDecrement.chromium.png|237| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverIncrement.chromium.png|768| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownDecrement.chromium.png|896| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverIncrement.chromium.png|237| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverInput.chromium.png|220| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverDecrement.chromium.png|273| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverIncrement.chromium.png|273| Changed|
|SpinButton Converged.Appearance- filled-darker.mouseDownIncrement.chromium.png|407| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownIncrement.chromium.png|495| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.focused.chromium.png|376| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownDecrement.chromium.png|230| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.rest.chromium.png|514| Changed|
|SpinButton Converged.Appearance- filled-lighter.hoverIncrement.chromium.png|40| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.rest.chromium.png|202| Changed|
|SpinButton Converged.Appearance- outline (default).rest.chromium.png|202| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownDecrement.chromium.png|492| Changed|
|SpinButton Converged.Appearance- outline (default).hoverDecrement.chromium.png|237| Changed|
|SpinButton Converged.Appearance- outline (default).hoverIncrement.chromium.png|237| Changed|
|SpinButton Converged.Appearance- outline (default).hoverInput.chromium.png|220| Changed|
|SpinButton Converged.Appearance- outline (default).mouseDownIncrement.chromium.png|495| Changed|
|SpinButton Converged.Appearance- outline (default).mouseDownDecrement.chromium.png|493| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverDecrement.chromium.png|629| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverInput.chromium.png|142| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.rest.chromium.png|512| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverIncrement.chromium.png|199| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.focused.chromium.png|187| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverIncrement.chromium.png|644| Changed|
|SpinButton Converged.Appearance- outline (default).focused.chromium.png|376| Changed|
|SpinButton Converged.Appearance- underline - RTL.focused.chromium.png|299| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverInput.chromium.png|183| Changed|
|SpinButton Converged.Appearance- underline - RTL.hoverDecrement.chromium.png|180| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownIncrement.chromium.png|162| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverDecrement.chromium.png|198| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownDecrement.chromium.png|759| Changed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownDecrement.chromium.png|423| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.rest.chromium.png|129| Changed|
|SpinButton Converged.Appearance- underline - RTL.hoverInput.chromium.png|163| Changed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownIncrement.chromium.png|428| Changed|
|SpinButton Converged.Appearance- underline.hoverInput.chromium.png|163| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownIncrement.chromium.png|790| Changed|
|SpinButton Converged.Appearance- underline - RTL.rest.chromium.png|155| Changed|
|SpinButton Converged.Appearance- underline - RTL.hoverIncrement.chromium.png|180| Changed|
|SpinButton Converged.Appearance- filled-lighter.focused.chromium.png|295| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverIncrement.chromium.png|248| Changed|
|SpinButton Converged.Appearance- underline.rest.chromium.png|155| Changed|
|SpinButton Converged.Appearance- underline.mouseDownDecrement.chromium.png|424| Changed|
|SpinButton Converged.Appearance- underline.focused.chromium.png|299| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.focused.chromium.png|329| Changed|
|SpinButton Converged.Appearance- underline.hoverDecrement.chromium.png|180| Changed|
|SpinButton Converged.Appearance- underline.mouseDownIncrement.chromium.png|429| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverIncrement.chromium.png|546| Changed|
|SpinButton Converged.At Max Bound - High Contrast.focused.chromium.png|501| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverInput.chromium.png|358| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownIncrement.chromium.png|224| Changed|
|SpinButton Converged.Appearance- filled-lighter.rest.chromium.png|23| Changed|
|SpinButton Converged.Appearance- underline.hoverIncrement.chromium.png|180| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.rest.chromium.png|186| Changed|
|SpinButton Converged.At Max Bound - RTL.focused.chromium.png|369| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownIncrement.chromium.png|689| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownDecrement.chromium.png|162| Changed|
|SpinButton Converged.At Max Bound - RTL.mouseDownIncrement.chromium.png|449| Changed|
|SpinButton Converged.At Max Bound - RTL.hoverIncrement.chromium.png|231| Changed|
|SpinButton Converged.At Max Bound.focused.chromium.png|369| Changed|
|SpinButton Converged.At Max Bound - High Contrast.rest.chromium.png|508| Changed|
|SpinButton Converged.At Max Bound - RTL.rest.chromium.png|195| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverDecrement.chromium.png|764| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownDecrement.chromium.png|892| Changed|
|SpinButton Converged.At Max Bound - RTL.hoverInput.chromium.png|214| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.rest.chromium.png|250| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverDecrement.chromium.png|270| Changed|
|SpinButton Converged.At Max Bound.hoverDecrement.chromium.png|233| Changed|
|SpinButton Converged.At Max Bound.hoverInput.chromium.png|214| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.focused.chromium.png|249| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.focused.chromium.png|248| Changed|
|SpinButton Converged.At Max Bound.rest.chromium.png|195| Changed|
|SpinButton Converged.At Min Bound - High Contrast.focused.chromium.png|501| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverDecrement.chromium.png|214| Changed|
|SpinButton Converged.At Max Bound - RTL.mouseDownDecrement.chromium.png|488| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.rest.chromium.png|185| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownIncrement.chromium.png|208| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverIncrement.chromium.png|236| Changed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownDecrement.chromium.png|501| Changed|
|SpinButton Converged.At Max Bound.mouseDownDecrement.chromium.png|489| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownDecrement.chromium.png|206| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverIncrement.chromium.png|576| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverInput.chromium.png|357| Changed|
|SpinButton Converged.At Min Bound - RTL.hoverDecrement.chromium.png|213| Changed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownIncrement.chromium.png|719| Changed|
|SpinButton Converged.At Min Bound - RTL.hoverIncrement.chromium.png|215| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownDecrement.chromium.png|227| Changed|
|SpinButton Converged.At Min Bound - RTL.hoverInput.chromium.png|213| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverInput.chromium.png|215| Changed|
|SpinButton Converged.At Max Bound - RTL.hoverDecrement.chromium.png|233| Changed|
|SpinButton Converged.At Min Bound - RTL.focused.chromium.png|368| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverDecrement.chromium.png|357| Changed|
|SpinButton Converged.At Min Bound - RTL.mouseDownDecrement.chromium.png|401| Changed|
|SpinButton Converged.At Min Bound.hoverDecrement.chromium.png|213| Changed|
|SpinButton Converged.At Min Bound - RTL.mouseDownIncrement.chromium.png|442| Changed|
|SpinButton Converged.At Min Bound - RTL.rest.chromium.png|194| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverInput.chromium.png|214| Changed|
|SpinButton Converged.At Min Bound.hoverIncrement.chromium.png|215| Changed|
|SpinButton Converged.At Min Bound.rest.chromium.png|194| Changed|
|SpinButton Converged.Disabled - Dark Mode.focused.chromium.png|53| Changed|
|SpinButton Converged.At Min Bound.hoverInput.chromium.png|213| Changed|
|SpinButton Converged.At Min Bound.mouseDownDecrement.chromium.png|401| Changed|
|SpinButton Converged.At Min Bound - High Contrast.rest.chromium.png|507| Changed|
|SpinButton Converged.At Min Bound.focused.chromium.png|368| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverIncrement.chromium.png|53| Changed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownDecrement.chromium.png|53| Changed|
|SpinButton Converged.Disabled - High Contrast.focused.chromium.png|338| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverDecrement.chromium.png|53| Changed|
|SpinButton Converged.Disabled - Dark Mode.rest.chromium.png|53| Changed|
|SpinButton Converged.Disabled - High Contrast.mouseDownIncrement.chromium.png|338| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverIncrement.chromium.png|338| Changed|
|SpinButton Converged.At Max Bound.hoverIncrement.chromium.png|231| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverInput.chromium.png|53| Changed|
|SpinButton Converged.Disabled - High Contrast.rest.chromium.png|338| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverInput.chromium.png|338| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverDecrement.chromium.png|338| Changed|
|SpinButton Converged.Disabled - High Contrast.mouseDownDecrement.chromium.png|338| Changed|
|SpinButton Converged.Disabled - RTL.focused.chromium.png|67| Changed|
|SpinButton Converged.Disabled - RTL.hoverInput.chromium.png|67| Changed|
|SpinButton Converged.Disabled - RTL.mouseDownDecrement.chromium.png|67| Changed|
|SpinButton Converged.Disabled.hoverDecrement.chromium.png|67| Changed|
|SpinButton Converged.Disabled.hoverInput.chromium.png|67| Changed|
|SpinButton Converged.Disabled - RTL.hoverDecrement.chromium.png|67| Changed|
|SpinButton Converged.Disabled.focused.chromium.png|67| Changed|
|SpinButton Converged.Disabled - RTL.mouseDownIncrement.chromium.png|67| Changed|
|SpinButton Converged.Disabled.mouseDownDecrement.chromium.png|67| Changed|
|SpinButton Converged.Disabled - RTL.rest.chromium.png|67| Changed|
|SpinButton Converged.At Min Bound.mouseDownIncrement.chromium.png|442| Changed|
|SpinButton Converged.Disabled - RTL.hoverIncrement.chromium.png|67| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverIncrement.chromium.png|283| Changed|
|SpinButton Converged.Disabled.rest.chromium.png|67| Changed|
|SpinButton Converged.Disabled.mouseDownIncrement.chromium.png|67| Changed|
|SpinButton Converged.Display Value - Dark Mode.rest.chromium.png|192| Changed|
|SpinButton Converged.Display Value - Dark Mode.focused.chromium.png|255| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverDecrement.chromium.png|283| Changed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownDecrement.chromium.png|243| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverInput.chromium.png|221| Changed|
|SpinButton Converged.Disabled.hoverIncrement.chromium.png|67| Changed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownIncrement.chromium.png|243| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverDecrement.chromium.png|755| Changed|
|SpinButton Converged.Display Value - High Contrast.mouseDownDecrement.chromium.png|884| Changed|
|SpinButton Converged.Display Value - High Contrast.mouseDownIncrement.chromium.png|899| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverInput.chromium.png|364| Changed|
|SpinButton Converged.Display Value - High Contrast.rest.chromium.png|514| Changed|
|SpinButton Converged.Display Value - RTL.mouseDownDecrement.chromium.png|498| Changed|
|SpinButton Converged.Display Value - RTL.hoverInput.chromium.png|220| Changed|
|SpinButton Converged.Display Value - High Contrast.focused.chromium.png|508| Changed|
|SpinButton Converged.Display Value - RTL.focused.chromium.png|376| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverIncrement.chromium.png|756| Changed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownIncrement.chromium.png|53| Changed|
|SpinButton Converged.Display Value - RTL.hoverDecrement.chromium.png|246| Changed|
|SpinButton Converged.Display Value - RTL.mouseDownIncrement.chromium.png|500| Changed|
|SpinButton Converged.Display Value - RTL.hoverIncrement.chromium.png|246| Changed|
|SpinButton Converged.Display Value - RTL.rest.chromium.png|202| Changed|
|SpinButton Converged.Display Value.focused.chromium.png|376| Changed|
|SpinButton Converged.Display Value.hoverDecrement.chromium.png|250| Changed|
|SpinButton Converged.Display Value.mouseDownDecrement.chromium.png|500| Changed|
|SpinButton Converged.Invalid- filled-darker.mouseDownDecrement.chromium.png|401| Changed|
|SpinButton Converged.At Max Bound.mouseDownIncrement.chromium.png|450| Changed|
|SpinButton Converged.Display Value.hoverInput.chromium.png|220| Changed|
|SpinButton Converged.Invalid- filled-darker.hoverDecrement.chromium.png|309| Changed|
|SpinButton Converged.Invalid- filled-darker.hoverInput.chromium.png|294| Changed|
|SpinButton Converged.Invalid- filled-darker.rest.chromium.png|294| Changed|
|SpinButton Converged.Display Value.hoverIncrement.chromium.png|250| Changed|
|SpinButton Converged.Invalid- filled-darker.mouseDownIncrement.chromium.png|407| Changed|
|SpinButton Converged.Invalid- filled-lighter.hoverIncrement.chromium.png|330| Changed|
|SpinButton Converged.Invalid- filled-darker.hoverIncrement.chromium.png|309| Changed|
|SpinButton Converged.Invalid- filled-lighter.focused.chromium.png|295| Changed|
|SpinButton Converged.Display Value.mouseDownIncrement.chromium.png|502| Changed|
|SpinButton Converged.Invalid- filled-darker.focused.chromium.png|277| Changed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownIncrement.chromium.png|425| Changed|
|SpinButton Converged.Invalid- outline.focused.chromium.png|376| Changed|
|SpinButton Converged.Invalid- filled-lighter.hoverInput.chromium.png|314| Changed|
|SpinButton Converged.Invalid- outline.hoverDecrement.chromium.png|330| Changed|
|SpinButton Converged.Invalid- filled-lighter.rest.chromium.png|314| Changed|
|SpinButton Converged.Invalid- underline.focused.chromium.png|299| Changed|
|SpinButton Converged.Invalid- underline.hoverDecrement.chromium.png|170| Changed|
|SpinButton Converged.Invalid- underline.hoverIncrement.chromium.png|170| Changed|
|SpinButton Converged.Invalid- outline.hoverIncrement.chromium.png|330| Changed|
|SpinButton Converged.Invalid- outline.hoverInput.chromium.png|314| Changed|
|SpinButton Converged.Invalid- filled-lighter.hoverDecrement.chromium.png|330| Changed|
|SpinButton Converged.Invalid- outline.rest.chromium.png|314| Changed|
|SpinButton Converged.Invalid- outline.mouseDownDecrement.chromium.png|493| Changed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownDecrement.chromium.png|420| Changed|
|SpinButton Converged.Display Value.rest.chromium.png|202| Changed|
|SpinButton Converged.Invalid- outline.mouseDownIncrement.chromium.png|495| Changed|
|SpinButton Converged.Invalid- underline.mouseDownDecrement.chromium.png|424| Changed|
|SpinButton Converged.Invalid- underline.mouseDownIncrement.chromium.png|429| Changed|
|SpinButton Converged.Invalid- underline.rest.chromium.png|153| Changed|
|SpinButton Converged.Size- medium (default) - RTL.default.chromium.png|202| Changed|
|SpinButton Converged.Size- medium (default).default.chromium.png|202| Changed|
|SpinButton Converged.Size- small - High Contrast.default.chromium.png|441| Changed|
|SpinButton Converged.Size- small - Dark Mode.default.chromium.png|166| Changed|
|SpinButton Converged.Size- small.default.chromium.png|175| Changed|
|SpinButton Converged.With appearance override.default.chromium.png|293| Changed|
|SpinButton Converged.Size- small - RTL.default.chromium.png|177| Changed|
|SpinButton Converged.Invalid- underline.hoverInput.chromium.png|153| Changed|
|SpinButton Converged.Custom Width - RTL.chromium.png|1953| Changed|
|SpinButton Converged.Custom Width - High Contrast.chromium.png|209| Changed|
|SpinButton Converged.Custom Width - Dark Mode.chromium.png|2986| Changed|
|SpinButton Converged.Custom Width.chromium.png|1953| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>